cmd/amp/function.go

```diff
@@ -26,6 +27,8 @@ var (
 	createFunctionCmd = &cobra.Command{
 		Use:   "create NAME IMAGE",
 		Short: "Create a function",
+		Long: `Create command registers a function into ETCD, with the specified name and image.
```

Comment:
The create command registers a function with the specified name and image. If successful, a function ID is returned.